capstone/src/main/resources/static/event.js

```diff
@@ -1,36 +1,144 @@
-  createEventAttribute(event.eventTitle, listEventElement);
-  createEventAttribute(event.eventDateTime, listEventElement);
-  createEventAttribute(event.eventDescription, listEventElement);
-  createEventAttribute(event.eventLatitude, listEventElement);
-  createEventAttribute(event.eventLongitude, listEventElement);
-  createEventAttribute(event.foodAvaliable, listEventElement);
-  createEventAttribute(event.requiredFee, listEventElement);
-  
-  return listEventElement;
+  // Location Using latitude as a filler until we finalize the location portion
+   const eventLocationElement = document.createElement('p');
+   eventLocationElement.innerText = event.eventLatitude;
+
+  // Organization
+  //const eventOrgElement = document.createElement('p');
+  //eventOrgElement.innerText = event.organization.name;
+
+  eventElement.appendChild(eventNameElement);
+  eventElement.appendChild(eventTimeElement);
+  //eventElement.appendChild(eventOrgElement);
+  if (displaySaveButton) {
+    eventElement.appendChild(createSaveEventButton(event));
+  }
+  return eventElement;
 }
 
-function createEventAttribute(attributeValue, listEventElement) {
-  const attributeValueElement = document.createElement('p');
-  attributeValueElement.innerText = attributeValue;
-  listEventElement.appendChild(attributeValueElement);
+/**
+ * Create event's review submission and format review listing
+ * @param event Event from Get call
+ * @return review section of event's listing
+ */
+function createReviewElement(event) {
+  const reviewElement = document.createElement('div');
+  reviewElement.className = 'reviews';
+
+  const reviewInputElement = document.createElement('input');
+  reviewInputElement.className = 'review-submission';
+  reviewInputElement.setAttribute('placeholder', 'Leave a review');
+  reviewInputElement.setAttribute('type', 'text');
+
+  reviewButtonElement = document.createElement('button');
+  reviewButtonElement.className = 'review-submission';
+  reviewButtonElement.innerText = 'Submit Review';
+  reviewButtonElement.addEventListener('click', () => {
+    if (reviewInputElement.value != '') {
+      newReview(event.datastoreID, reviewInputElement.value);
+    }
+  });
+
+  reviewElement.appendChild(reviewInputElement);
+  reviewElement.appendChild(reviewButtonElement);
+  reviewElement.appendChild(createReviewContainerElement(event.reviews));
+  return reviewElement;
+}
+
+/**
+ * Format each review to add to review container
+ * @param reviews List of reviews within Event object
+ * @return List of event's reviews to add to document
+ */
+function createReviewContainerElement(reviews) {
+  const reviewsContainer = document.createElement('div');
+
+  reviews.forEach((review) => {
+    const reviewContainer = document.createElement('div');
+    reviewContainer.className = 'review';
+
+    const reviewDetailsElement = document.createElement('div');
+    reviewDetailsElement.className = 'review-details';
```

Comment:
This gets a bit repetitive. If you offload it to a helper function that takes in element name, class name and inner text, and that returns the element reverence, most of these can be turned into a single-line:

` const reviewDetailsElement = createElement('div', 'review-details', ''));`

In principle, you could even pass an optional array of sub-elements.